connect/runtime/src/main/java/org/apache/kafka/connect/cli/ConnectDistributed.java

```diff
@@ -53,62 +53,25 @@
 public class ConnectDistributed {
     private static final Logger log = LoggerFactory.getLogger(ConnectDistributed.class);
 
-    public static void main(String[] args) throws Exception {
+    private Time time = Time.SYSTEM;
+    private long initStart = time.hiResClockMs();
```

Comment:
should this be `final` since it's already global? Are you planning to reassign it? (probably not given its name)